drivers/volume/portworx/portworx.go

```diff
@@ -2676,6 +2685,28 @@ func (d *portworx) ToggleCallHome(n node.Node, enabled bool) error {
 	return nil
 }
 
+func (d *portworx) ValidateStorageCluster() error {
+	pxOps, err := pxOperator.ListStorageClusters(schedops.PXNamespace)
```

Comment:
Instead of List we should just do Get, only 1 StorageCluster can be installed on k8s cluster.

Reply:
in torpedo we don't know the name of the operator, that's why we need to use list instead of getting by name